Jenkinsfile

```diff
@@ -263,6 +263,12 @@ node("ec2-fleet") {
                     )
             } catch(ex) {
                 sh "docker logs ${kbweb.id}"
+                withCredentials([[$class: 'StringBinding',
+                    credentialsId: 'SLACK_INTEGRATION_TOKEN',
+                    variable: 'SLACK_INTEGRATION_TOKEN',
+                ]]) {
+                        slackSend channel: "#github", color: "danger", message: "<${env.CHANGE_URL}|${env.CHANGE_TITLE}>\n:small_red_triangle: Test failed: <${env.BUILD_URL}|${env.JOB_NAME} ${env.BUILD_DISPLAY_NAME}> by ${env.CHANGE_AUTHOR}", teamDomain: "keybase", token: "${env.SLACK_INTEGRATION_TOKEN}"
```

Comment:
This won't work for master builds. Also, do we want to notify re: successes too? Finally, we might want a way to notify the change author, otherwise I'm not sure how much value there is in posting to slack.